README.md

```diff
@@ -56,7 +55,7 @@ $ node -v
 $ npm -v
 ```
 
-Install [Node](https://nodejs.org/en/download/) that includes npm in version 10.9.0 or later: 
+Install [Node](https://nodejs.org/en/download/) that includes npm in version 10.9.0 or later:
```

Comment:
With Angular 9, we need at least node version 12. Could we put all version information in one file? Otherwise this will be hard to maintain. Something like a version matrix like @subotic made for DSP.

Reply:
This change shouldn't be part of this PR. It should have been part of #63 !?

Reply:
Yes, you are right! Since it's already too late, we could do it with https://dasch.myjetbrains.com/youtrack/issue/DSP-185.

Reply:
I added it to https://dasch.myjetbrains.com/youtrack/issue/DSP-185.